### PR TITLE
docs(getting-started) Rework instructions for free trials

### PR DESCRIPTION
**Pull request by lena-larionova**
Based on issue https://konghq.atlassian.net/browse/DOCS-1388.

### Issue

The getting started guide does not sufficiently explain how to follow Admin API instructions and use provided endpoints for free trial users. This is a big problem, as users are constantly getting lost and having to go back and forth with support for the same questions, over and over.

### Changes
I tried out a couple of suggestions to restructure and clear up this content, including moving the instructions for setting up for free trials into [Expose your Services](https://docs.konghq.com/getting-started-guide/2.1.x/expose-services/). It ended up obscuring the content and not really fitting into the theme of the topic, so I left it in [Prepare to Administer Kong Gateway](https://docs.konghq.com/getting-started-guide/2.1.x/prepare/), but expanded the blue block into a full section. @connorckong hope this makes sense!

Details:
* Added callouts to Free Trials in the overview topic
* Removed giant blue panel from Prepare to Administer and reworked into two sections: 
  * Hosted Free Trial Endpoints: explained the two main endpoints they need for the getting started guide, provided examples
  * Create a User with an RBAC Token
* Added a blue callout to each topic where the Admin API is used, telling free trial users to set up their RBAC token first, as people could easily skip through content.

### Preview

[Overview](https://deploy-preview-2425--kongdocs.netlify.app/getting-started-guide/2.1.x/overview/#using-this-guide) - note in Using this Guide section
[Prepare to Administer](https://deploy-preview-2425--kongdocs.netlify.app/getting-started-guide/2.1.x/prepare) - the bulk of the content is here
The rest of the topics just have a note in them, eg: [Improve Performance](https://deploy-preview-2425--kongdocs.netlify.app/getting-started-guide/2.1.x/improve-performance/#set-up-the-proxy-caching-plugin)

**Note:** Most of the changes in Expose your Services are just linebreaks, feel free to ignore. Was originally going to work mostly in this topic.

### CODE REVIEW

File: app/getting-started-guide/2.1.x/overview.md [hunk 1]
```diff
@@ -88,6 +91,13 @@ place on the Control Plane.
 want to use HTTPie, install it from [here](https://httpie.org/).
 * Any references to “{{site.base_gateway}}” refer to features or concepts
 common to both {{site.ce_product_name}} and {{site.ee_gateway_name}}.
+* **Cloud-based Enterprise free trial users**: Since the {{site.ee_product_name}}
+ free trial is a Kong-hosted cloud trial, you'll need to refer to the generated
+ admin and proxy endpoints instead of defining your own, and you don't need to
+ use the port number in the URL. Head to
```
Comment (tlblessing) [inline app/getting-started-guide/2.1.x/overview.md:97]:
```suggestion
 use the port number in the URL. See
```

**Review by tlblessing — APPROVED** [review]
One minor suggested edit.

File: app/getting-started-guide/2.1.x/prepare.md [hunk 2]
```diff
@@ -13,30 +13,100 @@ Before you start this section, make sure that:
...
+  <img class="no-image-expand" src="/assets/images/icons/icn-enterprise-grey.svg"
+  alt="Enterprise" />
+  This section is only applicable to {{site.ee_product_name}} hosted free trials.
+  Skip to <a href="#verify-the-kong-gateway-configuration">Verify the Kong Gateway Configuration</a>
+  if you aren't a free trial user.
 </div>
 
+If you are trying out {{site.ee_product_name}} using a hosted free trial, you
+can go through this guide to experience some of this enhanced functionality.
+However, if you want to test out configuring {{site.ee_product_name}}
```
Comment (connorckong) [inline app/getting-started-guide/2.1.x/prepare.md:28]: We recommend that you get started using the Kong Manager user interface; however, if you want to test out configuring {{site.ee_product_name}}

**Review by connorckong — APPROVED** [review]
Left 1 very minor suggestion, but otherwise looks great!

**Pull request merged**